ocaml/alerts/expiry_alert.mli

```diff
@@ -0,0 +1,57 @@
+(*
+ * Copyright (C) Citrix Systems Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation; version 2.1 only. with the special
+ * exception on linking described in file LICENSE.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *)
+
+val update :
+     rpc:(Rpc.call -> Rpc.response)
+  -> session_id:[< `session] Ref.t
+  -> alert_obj_description:string
+  -> expired_message_id:string * int64
+  -> expiring_conditions:(int * (string * int64)) list
```

Comment:
Would suggest to add a comment to explain the meaning of the components in this type.
Does the `string * int64` refers same type of `expired_message_id`?

Reply:
Added comments in the header file: https://github.com/xapi-project/xen-api/pull/5060/files#diff-318524569501cd50cb93b08d9e5e86f2c11b3d61946f85b2ac38ea65ef68116e

> Does the string * int64 refers same type of expired_message_id?
Yes, and `expired_message_id` and `expiring_conditions` are merged into one list now.